app/Http/Controllers/Bible/BibleFileSetsController.php

```diff
@@ -397,19 +407,22 @@ private function showTextFilesetChapter(
             return $query->where('verse_end', '<=', $verse_end);
         })
         ->orderBy('verse_start')
-        ->select([
-            'bible_verses.book_id as book_id',
-            'books.name as book_name',
-            'books.protestant_order as book_order',
-            'bible_books.name as book_vernacular_name',
-            'bible_verses.chapter',
-            'bible_verses.verse_start',
-            'bible_verses.verse_end',
-            'bible_verses.verse_text',
-            'glyph_chapter.glyph as chapter_vernacular',
-            'glyph_start.glyph as verse_start_vernacular',
-            'glyph_end.glyph as verse_end_vernacular',
-        ])->orderBy('books.name', 'ASC');
+        ->orderBy('books.name', 'ASC')
+        ->orderBy('bible_verses.chapter');
```

Comment:
this orderBy clause is new, I think. I realize the bible_verses are out of order, but I question whether we want to order them and incur the extra cost

Reply:
I have added this new order clause because a test of the "[M] Bulk requests" group was not passing. I was debugging the issue and I figured out that the records were being returned sorted by chapter I guess it was related to way was built the query before.